src/lib.rs

```diff
@@ -184,6 +185,17 @@ fn is_isomorphic(
     Ok(res)
 }
 
+#[pyfunction]
```

Comment:
Could you add a docstring here to describe the functionality so it will be in the docs. You'll also have to add `retworkx.combine_graphs` to `docs/source/api.rst` for it to show up in the documentation.